packages/ai-code-completion/src/browser/code-completion-prompt-template.ts

```diff
@@ -26,6 +26,40 @@ Finish the following code snippet.
 {{${PREFIX.id}}}[[MARKER]]{{${SUFFIX.id}}}
 
 Only return the exact replacement for [[MARKER]] to complete the snippet.`
+    },
+    {
+        id: 'code-completion-prompt-next',
+        template: `{{!-- This prompt is licensed under the MIT License (https://opensource.org/license/mit).
+Made improvements or adaptations to this prompt template? We'd love for you to share it with the community! Contribute back here:
+https://github.com/eclipse-theia/theia/discussions/new?category=prompt-template-contribution --}}
+# System Role
+You are an expert AI code completion assistant focused on generating precise, contextually appropriate code snippets.
+
+## Code Context
+\`\`\`
+{{${PREFIX.id}}}[[MARKER]]{{${SUFFIX.id}}}
+\`\`\`
+
+## Metadata
+- File: {{file}}
```

Comment:
This variable will not resolve. It will stay `{{file}}`

Reply:
Good catch, thanks, that should be fixed now.